### ISSUE TITLE
Support export of schema.org JSON-LD formats

### ISSUE THREAD

**Issue by ronaldtse** [glossarist/glossarist-desktop#155]
https://schema.org/DefinedTerm
https://schema.org/DefinedTermSet

Schema.org supports the DefinedTerm and DefinedTermSet structured data objects.

DefinedTerm sample:
```json
{
 "@context": "https://schema.org/",
 "@type": "DefinedTerm",
 "name": "Shoe Machine Operators and Tenders",
 "termCode": "51-6042.00",
 "description": "Operate or tend a variety of machines to join, decorate, reinforce, or finish shoes and shoe parts.",
 "inDefinedTermSet": "http://onetonline.org"
}
```

DefinedTermSet sample:
```json
[
        {
                "@context": "https://schema.org/"
        },
        {
                "@type": ["DefinedTermSet","Book"],
                "@id": "http://openjurist.org/dictionary/Ballentine",
                "name": "Ballentine's Law Dictionary"
        },
        {
                "@type": "DefinedTerm",
                "@id": "http://openjurist.org/dictionary/Ballentine/term/calendar-year",
                "name": "calendar year",
                "description": "The period from January 1st to December 31st, inclusive, of any year.",
                "inDefinedTermSet": "http://openjurist.org/dictionary/Ballentine"
        },
        {
                "@type": "DefinedTerm",
                "@id": "http://openjurist.org/dictionary/Ballentine/term/schema",
                "name": "schema",
                "description": "A representation of a plan or theory in the form of an outline or model.",
                "inDefinedTermSet": "http://openjurist.org/dictionary/Ballentine"
        }
]
```

We could:
* Export schema.org formats of individual terms
* Include such export inside the generated HTML

**Comment by strogonoff** [comment]
This should be another export format in addition to RDF, right?￼

**Comment by ronaldtse** [comment]
Yes. Schema.org data structures can also be represented in RDF but these two objects (DefinedTerm, DefinedTermSet) only have examples available in JSON-LD. These are not "final" schema.org structures.

DefinedTerm corresponds to a single concept in a specific language

field name | description
-- | --
inDefinedTermSet | URL of collection
termCode | human-readable ID of concept/term
additionalType | null
alternateName | synonyms
description | definition + Notes/Examples
disambiguatingDescription
identifier | URN/ID of concept/term
image | image for concept
mainEntityOfPage | URL of term (applicable when used outside of site)
name | designation of concept
potentialAction | null
sameAs | relations for "identical"
subjectOf | URL of collection
url | URL of concept


DefinedTermSet corresponds to a single "concept register" of a specific language

field name | description
-- | --
hasDefinedTerm | the DefinedTerm data
name | Name of collection
abstract | Description
acquireLicensePage | URL to copyright page
author | URL to Author/About page
citation | Some text about citation
copyrightHolder | Author information
copyrightNotice | URL to copyright page
copyrightYear	| year
dateCreated | DateTime string of creation
dateModified | DateTime string of last modified
datePublished | DateTime string of publication
inLanguage | BCP47 language code
license	| URL to license
publisher	| Publisher
usageInfo	| URL to about page
